prometheus/src/test/scala/filodb/prometheus/parse/ParserSpec.scala

```diff
@@ -678,6 +678,7 @@ class ParserSpec extends AnyFunSpec with Matchers {
       //if(lp.toString != e) {
```

Comment:
Can remove dead code?

Reply:
hm, that's not part of my change, though I do see that there an extra line added next to it.